### PR TITLE
Add missing "Open..." file menu

### PR DESCRIPTION
**Pull request by martinRenou**
Since Notebook 7, the `Open...` menu disappeared, this PR brings it back.

Notebook <=6 behavior:
https://github.com/jupyter/notebook/assets/21197331/89f09978-3338-4d3f-a913-592dceab9d92

Notebook 7 with this PR:
https://github.com/jupyter/notebook/assets/21197331/4acc8da0-c351-4118-9ac4-38b0bb4d1fe9

### CODE REVIEW

**Comment by martinRenou** [comment]
Bot update playwright snapshots

File: packages/notebook-extension/src/index.ts [hunk 1]
```diff
@@ -177,6 +177,40 @@ const closeTab: JupyterFrontEndPlugin<void> = {
...
+    menu: IMainMenu,
+    translator: ITranslator | null
+  ) => {
+    const { commands } = app;
+    translator = translator ?? nullTranslator;
+    const trans = translator.load('notebook');
+
+    const id = 'notebook:open-tree-tab';
+    commands.addCommand(id, {
+      label: trans.__('Open...'),
```
Comment (krassowski) [inline packages/notebook-extension/src/index.ts:201]:
```suggestion
      label: trans.__('Open…'),
```

Comment (martinRenou) [reply packages/notebook-extension/src/index.ts]: Thanks!

**Comment by martinRenou** [comment]
Trigger CI

**Comment by jtpio** [comment]
These workflow failures (cancelled) are also happening on other PRs, and pass after restarting them, not sure why:

image

File: ui-tests/test/mobile.spec.ts-snapshots/notebook-firefox-linux.png
Comment (jtpio) [inline ui-tests/test/mobile.spec.ts-snapshots/notebook-firefox-linux.png:1]: This one is likely flaky:

https://github.com/jupyter/notebook/issues/7370#issuecomment-2128791005

**Review by jtpio — APPROVED** [review]
Thanks!

**Pull request merged**